UX4_ValidityReliability.ipynb

```diff
@@ -372,14 +373,14 @@
     }
    ],
    "source": [
-    "#find t-statsistic for critical value of upper tail where p=0.025\n",
+    "# Find t-statsistic for critical value of upper tail where p = 0.025\n",
```

Comment:
"p" has never been introduced. Shouldn't this be "(alpha level) of 0.05" (same as previous code block)?